File: answers/javascript/utilityfunctions/testforxor.js
module.exports = function testForXOR( stringA, stringB ) {
  const results = [];

  for(let i=0; i< stringA.length; i++) {
    if( stringA[i] === stringB[i] ) {
      results.push( '0' );
    } else {
      results.push( '1' );
    }
  }

  return results.join('');
};

File: answers/attempt_2/python/utility/test_for_xor.py
def test_for_xor( string_a, string_b ):
  results = []

  for i in range( len( string_a ) ):
    if string_a[ i ] == string_b[ i ]:
      results.append( '0' )
    else:
      results.append( '1' )

  return ''.join( results )